lib/API/JSON/v1/SendingTaskSubscribers.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace MailPoet\API\JSON\v1;
+
+use MailPoet\Listing;
+use MailPoet\Cron\CronHelper;
+use MailPoet\Models\Newsletter;
+use MailPoet\Models\ScheduledTask;
+use MailPoet\Config\AccessControl;
+use MailPoet\API\JSON\Error as APIError;
+use MailPoet\Settings\SettingsController;
+use MailPoet\WP\Functions as WPFunctions;
+use MailPoet\Models\ScheduledTaskSubscriber;
+use MailPoet\API\JSON\Endpoint as APIEndpoint;
+use MailPoet\Models\SendingQueue as SendingQueueModel;
+
+if (!defined('ABSPATH')) exit;
+
+class SendingTaskSubscribers extends APIEndpoint {
+  public $permissions = [
+    'global' => AccessControl::PERMISSION_MANAGE_EMAILS,
+  ];
+
+  /** @var Listing\Handler */
+  private $listing_handler;
+
+  /** @var SettingsController */
+  private $settings;
+
+  /** @var WPFunctions */
+  private $wp;
+
+  function __construct(
+    Listing\Handler $listing_handler,
+    SettingsController $settings,
+    WPFunctions $wp
+  ) {
+    $this->listing_handler = $listing_handler;
+    $this->settings = $settings;
+    $this->wp = $wp;
+  }
+
+  function listing($data = []) {
+    $newsletter_id = !empty($data['params']['id']) ? (int)$data['params']['id'] : false;
+    $tasks_ids = SendingQueueModel::select('task_id')
+      ->where('newsletter_id', $newsletter_id)
+      ->findArray();
+    if (empty($tasks_ids)) {
+      return $this->errorResponse([
+        APIError::NOT_FOUND => __('This newsletter is not being sent to any subcriber yet.', 'mailpoet'),
+      ]);
+    }
+    $data['params']['task_ids'] = array_map(function($item) {
```

Comment:
We could use `array_column` here, but since it is such a broken function, I don't know if `array_map` isn't better here...

Reply:
I think `array_column` makes more sense in that case. At least it's less verbose.